### PR TITLE
add support for sub sections

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn review plaese

File: molo/core/models.py [hunk 1]
```diff
@@ -40,14 +43,24 @@ class Meta:
...
 
-    subpage_types = ['core.ArticlePage']
+    subpage_types = ['core.ArticlePage', 'core.SectionPage']
     search_fields = Page.search_fields + (
         index.SearchField('description'),
     )
 
     def articles(self):
-        return ArticlePage.objects.live().descendant_of(self)
+        return ArticlePage.objects.live().child_of(self)
```
Comment (smn) [inline molo/core/models.py]: What's the difference between these two?

Comment (smn) [inline molo/core/models.py]: Hierarchy?

Comment (smn) [inline molo/core/models.py]: Ah! Should've realised this earlier, direct children vs descendants.

**Comment by smn** [comment]
:+1:

**Pull request merged**